microprofile/WFLY-14281_Integrate-MP-Fault-Tolerance-3.0.adoc

```diff
@@ -0,0 +1,86 @@
+= Integrate MP Fault Tolerance 3.0 into EAP
+:author:            Jason Lee
+:email:             jasondlee@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+MicroProfile 4.0 provides a major upgrade for https://github.com/eclipse/microprofile-fault-tolerance/releases/tag/3.0[MicroProfile Fault Tolerance 3.0] with changes described in its
+https://download.eclipse.org/microprofile/microprofile-fault-tolerance-3.0/microprofile-fault-tolerance-spec-3.0.html#release_notes_30[Release Notes].
+
+This component upgrade also covers the upgrade of SmallRye Fault Tolerance that implements MicroProfile Fault Tolerance 3.0.
+
+== Issue Metadata
+ 
+=== Issue
+
+* https://issues.jboss.org/browse/WFLY-14281[WFLY-14281 Integrate MP Fault Tolerance 3.0]
+
+=== Related Issues
+
+* https://issues.redhat.com/browse/EAP7-1594[EAP7-1594 Integrate MP Fault Tolerance 3.0 into EAP]
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+* mailto:rhusar@redhat.com[Radoslav Husar]
+
+=== QE Contacts
+
+* mailto:fburzigo@redhat.com[Fabio Burzigotti]
+
+=== Testing By
+
+* [x] Engineering
+
+* [ ] QE
+
+=== Affected Projects or Components
+
+* WildFly
+
+=== Other Interested Projects
+
+* MicroProfile
+
+== Requirements
+
+* Upgrade MicroProfile Fault Tolerance (specification) to 3.0
+* Upgrade smallrye-fault-tolerance to version 5.x (which implements the above specification)
+* Run all MicroProfile Fault Tolerance tests in WildFly integration test suite
+* Run the Eclipse MicroProfile Fault Tolerance TCK without failures
+
+=== Hard Requirements
+
+* Artifact versions must be upgraded
+* Existing tests must be updated as needed and run cleanly
+
+=== Nice-to-Have Requirements
+
+=== Non-Requirements
+
+== Test Plan
+
+* Run existing tests (part of the CI runs):
+** `wildfly/testsuite/integration/microprofile`
+** `wildfly/testsuite/integration/microprofile-tck/fault-tolerance`
+
```

Comment:
See my comments above regarding testing the integration between Metrics and FT.

Reply:
Those are covered.

Reply:
Ok, thanks